build/pgbouncer/Dockerfile

```diff
@@ -2,13 +2,15 @@ ARG BASEOS
 ARG BASEVER
 ARG PG_FULL
 ARG PREFIX
-FROM ${PREFIX}/crunchy-pg-base:${BASEOS}-${PG_FULL}-${BASEVER}
+FROM ${PREFIX}/crunchy-base:${BASEOS}-${PG_FULL}-${BASEVER}
 
 # For RHEL8 all arguments used in main code has to be specified after FROM
 ARG DFSET
 ARG PACKAGER
 
 ARG PG_MAJOR
+# Needed due to lack of environment substitution trick on ADD
+ARG PG_LBL
```

Comment:
Why is this needed? I don't see it being used.

Reply:
same as above